.github/CHANGELOG.md

```diff
@@ -47,59 +47,61 @@
-  A = MMTensor(np.random.rand(2, 3, 4), axis_labels=["foo", "bar", "contract"])
-  B = MMTensor(np.random.rand(4, 5, 6), axis_labels=["contract", "baz", "qux"])
+    ```python
+    from mrmustard.math.mmtensor import MMTensor
 
-  # perform a tensor contraction
-  C = A @ B
-  C.axis_labels  # ["foo", "bar", "baz", "qux"]
-  C.shape # (2, 3, 5, 6)
-  C.tensor # extract actual result
-  ```
+    # define two tensors
+    A = MMTensor(np.random.rand(2, 3, 4), axis_labels=["foo", "bar", "contract"])
+    B = MMTensor(np.random.rand(4, 5, 6), axis_labels=["contract", "baz", "qux"])
 
-* MrMustard's settings object (accessible via `from mrmustard import settings`) now supports `SEED` (an int).
-  This will give reproducible results whenever randomness is involved. The seed is unset by default,
-  and it can be unset again with `settings.SEED = None`. If one desires,
-  the seeded random number generator is accessible directly via `settings.rng` (e.g. `settings.rng.normal()`).
-  [(#183)](https://github.com/XanaduAI/MrMustard/pull/183)
+    # perform a tensor contraction
+    C = A @ B
+    C.axis_labels  # ["foo", "bar", "baz", "qux"]
+    C.shape # (2, 3, 5, 6)
+    C.tensor # extract actual result
+    ```
 
-* The `Circuit` class now has an ascii representation, which can be accessed via the repr method.
-  It looks great in Jupyter notebooks! There is a new option at `settings.CIRCUIT_DECIMALS` which controls
-  the number of decimals shown in the ascii representation. If None only the name of the gate is shown.
-  [(#196)](https://github.com/XanaduAI/MrMustard/pull/196)
+  * MrMustard's settings object (accessible via `from mrmustard import settings`) now supports `SEED` (an int).
+    This will give reproducible results whenever randomness is involved. The seed is unset by default,
+    and it can be unset again with `settings.SEED = None`. If one desires,
+    the seeded random number generator is accessible directly via `settings.rng` (e.g. `settings.rng.normal()`).
+    [(#183)](https://github.com/XanaduAI/MrMustard/pull/183)
+
+  * The `Circuit` class now has an ascii representation, which can be accessed via the repr method.
+    It looks great in Jupyter notebooks! There is a new option at `settings.CIRCUIT_DECIMALS` which controls
+    the number of decimals shown in the ascii representation. If None only the name of the gate is shown.
+    [(#196)](https://github.com/XanaduAI/MrMustard/pull/196)
+  * PNR sampling from Gaussian circuits using density matrices can now be performed faster. When all modes are detected, this is done by replacing `math.hermite_renormalized` by `math.hermite_renormalized_diagonal`. In case all but the first mode are detected, `math.hermite_renormalized_1leftoverMode` can be used. The complexity of these new methods is equal to performing a pure state simulation. The methods are differentiable, such that they can be used for defining a costfunction. [(#154)](https://github.com/XanaduAI/MrMustard/pull/154)
```

Comment:
```suggestion

  * PNR sampling from Gaussian circuits using density matrices can now be performed faster. When all modes are detected, this is done by replacing `math.hermite_renormalized` by `math.hermite_renormalized_diagonal`. In case all but the first mode are detected, `math.hermite_renormalized_1leftoverMode` can be used. The complexity of these new methods is equal to performing a pure state simulation. The methods are differentiable, such that they can be used for defining a costfunction. [(#154)](https://github.com/XanaduAI/MrMustard/pull/154)
```